Dockerfile

```diff
@@ -1,114 +1,25 @@
-FROM ubuntu:18.04
-ARG DEBIAN_FRONTEND=noninteractive
+FROM rocker/geospatial:4.1.0
```

Comment:
what is the advantage of starting with this docker image?

Reply:
Using the rocker/geospatial image grossly simplifies the docker image, and cuts the image build time to only a few minutes. It's the official R docker image so it should be readily available and stable. 

The main simplifications come from the fact that the following packages are already installed
- R
- gdal and other geospatial packages
- a large number of the required R packages
- ncdf4 (which requires the user to download a tarball)